snuba/query/processors/mapping_optimizer.py

```diff
@@ -83,6 +97,33 @@ def __init__(self, column_name: str, hash_map_name: str, killswitch: str) -> Non
                 Param("right_hand_side", Literal(Any(str))),
             ),
         )
+        self.__tag_exists_patterns = [
+            FunctionCall(
```

Comment:
Note: it may be a better idea to run the EmptyTagProcessor as part of this processor so we only have to worry about/match one case. Thoughts?

Reply:
I have mixed feelings about it. On the one hand it makes sense because these processors are working on similar things, but on the other hand I feel like we should be striving to keep processors as simple as possible for readability and debugging.